### ISSUE TITLE
Uncaught Error: ENOENT, open 'c:\Users\Ralitchka\Documents\OSU\CS 290\assignment1\bio.md'

### ISSUE THREAD

**Issue by abadjier** [atom/atom#4998]
[Enter steps to reproduce below:]

1. ...Create a branch of the second to last commit of the master
2. ...Switch to that branch
3. Made some changes to file, commit changes
4. Switch back to master 
5. Atom updated to the "Master version" and resulted in the problem below:

**Atom Version**: 0.169.0
**System**: Microsoft Windows 7 Home Premium 
**Thrown From**: Atom Core

### Stack Trace

Uncaught Error: ENOENT, open 'c:\Users\Ralitchka\Documents\OSU\CS 290\assignment1\bio.md'

```
At c:\Users\Ralitchka\AppData\Local\atom\app-0.169.0\resources\app\node_modules\q\q.js:126

Error: ENOENT, open 'c:\Users\Ralitchka\Documents\OSU\CS 290\assignment1\bio.md'
  at Error (native)

```

### Commands

```
  2x -2:27.8 editor:newline (atom-text-editor.editor)
 14x -1:18.4 core:move-left (atom-text-editor.editor)
     -1:14.7 core:backspace (atom-text-editor.editor)
     -1:13.5 core:delete (atom-text-editor.editor)
     -1:11.3 editor:move-to-end-of-screen-line (atom-text-editor.editor)
  9x -1:10.3 core:backspace (atom-text-editor.editor)
 14x -0:53.3 core:move-left (atom-text-editor.editor)
     -0:51.0 editor:newline (atom-text-editor.editor)
     -0:45.3 core:delete (atom-text-editor.editor)
  4x -0:42.6 core:backspace (atom-text-editor.editor)
     -0:00.0 core:save (atom-text-editor.editor)
```

### Config

```json
{
  "core": {},
  "editor": {}
}
```

### Installed Packages

```coffee
# User
No installed packages

# Dev
No dev packages
```

**Comment by sWski** [comment]
I have similar issue. I reverted changes in the file via GitHub for Windows, then switched to Atom.

1. Make some changes and save the file.
2. Revert these changes via git.
3. Switch back to Atom.

**Atom Version**: 0.169.0
**System**: Microsoft Windows 7 Professional 
**Thrown From**: Atom Core

### Stack Trace

Uncaught Error: ENOENT, open 'c:\Dropbox\Projects\Recipes\static\css\style.css'

```
At c:\Users\jakub.MII\AppData\Local\atom\app-0.169.0\resources\app\node_modules\q\q.js:126

Error: ENOENT, open 'c:\Dropbox\Projects\Recipes\static\css\style.css'
  at Error (native)

```

### Commands

```
     -0:07.9 editor:toggle-line-comments (atom-text-editor.editor)
     -0:07.6 core:save (atom-text-editor.editor)
     -0:00.4 editor:toggle-line-comments (atom-text-editor.editor)
     -0:00.0 core:save (atom-text-editor.editor)
```

### Config

```json
{
  "core": {
    "disabledPackages": [
      "background-tips",
      "html-id-class-snippets",
      "open-on-github",
      "package-generator",
      "spell-check"
    ],
    "themes": [
      "seti-ui",
      "seti-monokai"
    ]
  },
  "editor": {
    "fontSize": 17,
    "showIndentGuide": true,
    "tabLength": 4,
    "softWrap": true,
    "invisibles": {}
  }
}
```

### Installed Packages

```coffee
# User
atom-jinja2, v0.5.0
autoclose-html, v0.13.0
color-picker, v1.2.6
html-helper, v0.2.3
minimap, v3.5.4
seti-monokai, v0.3.1
seti-ui, v0.6.1

# Dev
No dev packages
```

**Comment by kevinsawicki** [comment]
I believe this is a duplicate of https://github.com/atom/atom/issues/4850 and will be fixed by #4906
